components/brave_wallet_ui/constants/types.ts

```diff
@@ -1071,7 +1077,7 @@ export interface NavOption {
   id: NavIDTypes
   name: string
   icon: string
-  route: WalletRoutes
+  route: WalletRoutes | typeof AccountPageTabs[keyof typeof AccountPageTabs]
```

Comment:
nit: let's create/export a type-alias for this
`export type AccountPageTabs = typeof AccountPageTabs[keyof typeof AccountPageTabs]`